packages/shared/src/hooks/post/useSmartTitle.ts

```diff
@@ -31,20 +33,28 @@ export const useSmartTitle = (post: Post): UseSmartTitle => {
   const { isPlus } = usePlusSubscription();
   const { completeAction } = useActions();
   const { flags } = useSettingsContext();
+  const { queryKey } = useActiveFeedContext();
 
   const { clickbaitShieldEnabled } = flags || {};
 
   const key = useMemo(
     () => [...getPostByIdKey(post?.id), { key: 'title' }],
     [post?.id],
   );
+
+  const { fetchTranslations } = useTranslation({
+    queryKey: queryKey || getPostByIdKey(post?.id),
+    queryType: queryKey ? 'feed' : 'post',
```

Comment:
We want to update `translation` for currently shown post that is passed as a prop to `useSmartTitle` so if we are in feed I use feed mode and else just single post